c_sharp/image_classification/model/augment_model.py

```diff
@@ -0,0 +1,20 @@
+import onnx
+import torch
+from onnxruntime_extensions import pnp
+
+mobilenet_model = onnx.load_model('mobilenetv2-12.onnx')
+augmented_model = pnp.SequentialProcessingModule(
+    pnp.PreMobileNet(224),
+    mobilenet_model,
+    pnp.PostMobileNet())
+
+# The image size is dynamic, the 400x500 here is to get a fake input to enable export
+fake_image_input = torch.ones(500, 400, 3).to(torch.uint8)
+augmented_model.forward(fake_image_input)
```

Comment:
This is slightly misleading as it's not channels last, Something like 3, 500, 400 would be clearer.